src/CommerceModel.BetterRetail/artifacts/OOE/BetterRetail/QueueImportProfileSchemaActivity/groups.json

```diff
@@ -0,0 +1,11 @@
+﻿[
+  {
+    "displayOrder": 1,
```

Comment:
Since it is the only element here for now, would it be better to update this to 0?